webpack.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+	entry: "./client/js/lounge.js",
+	resolve: {
+		extensions: ["", ".js"]
```

Comment:
As per our discussion on IRC, could you remove that section altogether and rely on [the defaults](https://webpack.github.io/docs/configuration.html#resolve-extensions)?